ts/Series/AreaRange/AreaRangeDataLabelsOptionsObject.d.ts

```diff
@@ -0,0 +1,37 @@
+/* *
+ *
+ *  (c) 2010-2020 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type DataLabelOptions from '../../Core/Series/DataLabelOptions';
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+export interface AreaRangeDataLabelsOptionsObject extends DataLabelOptions {
```

Comment:
This all (interface, file name, import, export) should become `AreaRangeDataLabelOptions`.